Classes/Command/FluidTemplatePrecompileCommand.php

```diff
@@ -0,0 +1,161 @@
+ */
+class FluidTemplatePrecompileCommand extends Command
+{
+    use FluidPrecompilerServiceTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Precompile a Fluid resource')
+            ->addArgument(
+                'extension',
+                InputArgument::OPTIONAL,
+                'Extension key of which the Fluid resources should be compiled. Leave empty to compile all',
+                null
+            )
+            ->addOption(
+                'fail',
+                null,
+                InputOption::VALUE_NONE,
+                'Fail with a non-zero return code if any template could not be compiled'
+            )
+            ->addOption(
+                'show-only-failed',
+                null,
+                InputOption::VALUE_NONE,
+                'Show only failed Fluid templates which failed to compile (in conjunction with -v)'
+            )
+            ->addOption(
+                'limit',
+                'l',
+                InputOption::VALUE_REQUIRED,
+                'Limit template information shown of each extension (in conjunction with -v)',
+                5
+            )
+            ->addOption(
+                'disable-error-handler',
+                null,
+                InputOption::VALUE_NONE,
+                'Temporarily disable any registered error handler for compiling'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $outputHelper = new SymfonyStyle($input, $output);
+
+        // Get arguments and options
+        $extension = $input->getArgument('extension');
+        $fail = $input->getOption('fail');
+        $onlyFailed = $input->getOption('show-only-failed');
+        $limit = $input->getOption('limit');
+        $disableErrorHandlers = $input->getOption('disable-error-handler');
+
+        // Test exception handlers
+        if (set_error_handler(function() {}) && !$disableErrorHandlers) {
+            $outputHelper->warning(
+                "\nIt seems like there is an error handler registered."
+                . "\nIn some cases this can prevent the precompiling to run properly."
+                . "\nAn ErrorHandler might raise an exception for a PHP Warning."
+            );
+
+            restore_error_handler();
+        }
+
+        $result = $this->getPrecompilerService()->warmup($extension);
+
+        // General information
+        $templatesCount = 0;
+        foreach ($result as $extensionResult) {
+            $templatesCount += count($extensionResult);
+        }
+
+        $uncompilable = array_column($result, 'uncompilable');
+        $uncompilable = array_sum($uncompilable);
+
+        $outputHelper->section('General');
+        $outputHelper->text(vsprintf('Compiled %d extension(s) with %d template(s) in total.', [count($result), $templatesCount]));
```

Comment:
A more appropriate error message would be `Compiled %d template(s) in %d extension(s)`.